dtl/tag/logic.js

```diff
@@ -172,10 +172,13 @@ define([
 							zipped[assign[k]] = item[k];
 						}
 						lang.mixin(context, zipped);
-					}else{
+					} else{
```

Comment:
Dojo 1.x actually favors `}else{` (as wrong as that feels today). Other than that (and one other instance), this change looks fine, thanks!